<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–ú–µ–Ω–µ–¥–∂–µ—Ä –∞–≤—Ç–æ–º–æ–±—ñ–ª—ñ–≤</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="topControls">
      <item>
       <widget class="QLineEdit" name="searchText">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>‚ûï –î–æ–¥–∞—Ç–∏</string>
        </property>
        <property name="toolTip">
         <string>–î–æ–¥–∞—Ç–∏ –Ω–æ–≤–∏–π –∞–≤—Ç–æ–º–æ–±—ñ–ª—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text">
         <string>‚úèÔ∏è –†–µ–¥–∞–≥—É–≤–∞—Ç–∏</string>
        </property>
        <property name="toolTip">
         <string>–†–µ–¥–∞–≥—É–≤–∞—Ç–∏ –æ–±—Ä–∞–Ω–∏–π –∞–≤—Ç–æ–º–æ–±—ñ–ª—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>üóëÔ∏è –í–∏–¥–∞–ª–∏—Ç–∏</string>
        </property>
        <property name="toolTip">
         <string>–í–∏–¥–∞–ª–∏—Ç–∏ –æ–±—Ä–∞–Ω–∏–π –∞–≤—Ç–æ–º–æ–±—ñ–ª—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadButton">
        <property name="text">
         <string>üìÇ –ó–∞–≤–∞–Ω—Ç–∞–∂–∏—Ç–∏</string>
        </property>
        <property name="toolTip">
         <string>–ó–∞–≤–∞–Ω—Ç–∞–∂–∏—Ç–∏ –¥–∞–Ω—ñ –∑ —Ñ–∞–π–ª—É</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>üíæ –ó–±–µ—Ä–µ–≥—Ç–∏</string>
        </property>
        <property name="toolTip">
         <string>–ó–±–µ—Ä–µ–≥—Ç–∏ –¥–∞–Ω—ñ —É —Ñ–∞–π–ª</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="operationsButton">
        <property name="text">
         <string>‚öôÔ∏è –û–ø–µ—Ä–∞—Ü—ñ—ó</string>
        </property>
        <property name="toolTip">
         <string>–í—ñ–¥–∫—Ä–∏—Ç–∏ –≤—ñ–∫–Ω–æ –æ–ø–µ—Ä–∞—Ü—ñ–π –∑ –¥–∞–Ω–∏–º–∏</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>